### ISSUE TITLE
periodic public database dumps are failing

### ISSUE THREAD

**Issue by mhl** [mysociety/pombola#2317]
Since the databases were migrated to the new server (pelican) we're getting a version mismatch error when the cron job on pluto runs to try to create the publicly available database dump:

```
pg_dump: server version: 9.6.4; pg_dump version: 9.1.24lts2
pg_dump: aborting because of server version mismatch
```

(Full error email here: https://groups.google.com/a/mysociety.org/forum/#!topic/cron-pombola/TDWboYkWF-c )

The simplest solution (as opposed, say, to getting a later `pg_dump` client installed) might be to redeploy those Pombola instances (+ their associated staging sites) to one of the new stretch-based application servers, since that would be a Good Thing to do anyway. /cc @sagepe

**Comment by dracos** [comment]
Or run pg_dump on the server rather than the client? See e.g. https://github.com/mysociety/misc-scripts/pull/29

**Comment by mhl** [comment]
Good idea. The command here: https://github.com/mysociety/pombola/blob/790cd5354c124c7f7224b50c129d30be11685ab1/pombola/core/management/commands/core_database_dump.py#L222-L226 would need to be changed to start `'ssh', db_settings['HOST']` but otherwise I think that should Just Work™

**Comment by mhl** [comment]
Hopefully I've done this correctly in https://github.com/mysociety/pombola/commit/6ec51002c51239e89c4d0a7a3ae106307602258c

... but we can see tomorrow if that has worked properly. (It was fine for the Mzalendo staging site.)

An extra step I'd like to do before closing this issue is to add a `command` restriction to the authorized keys file of the site user on the database server, since I had to add passwordless ssh login for the site user to the db server for this to work, and I'd be more comfortable with that if only the `pg_dump` command could be run.
